### ISSUE TITLE
Issue: Lines 76 and 77 fail for $PythonExecutable value containing non-alphanumeric characters like ), ( etc..

### ISSUE THREAD

**Issue by vcsekhar** [oracle/oci-cli#53]
Issue: Lines 76 and 77 fail for $PythonExecutable value containing non-alphanumeric characters like ), ( etc..

Solution that worked for me:

Lines 76 and 77 to be changed to :

```# If you prefer to use $EscapedExecutable through out, then 
  $EscapedExecutable = $PythonExecutable 
  $PythonVersion = Invoke-Expression "& `"$EscapedExecutable`" -c 'import platform;print(platform.python_version())'"
  ```

Explanation:
   Instead of trying to escape just spaces (like the current logic), make powershell to call/invoke (using & ) a not-escaped value as is by enclosing it in between ` `.

I forked the install.ps1 with my fix included for your review-and-merge-easiness.

**Comment by smak-oracle** [comment]
thanks your suggestion and fix. we will check it out and merge it  after tested

**Comment by vish1** [comment]
@vcsekhar, thank you for the suggested changes. The fix is included in the 2.4.24 release.
